src/MSBuild/app.amd64.config

```diff
@@ -78,14 +78,14 @@
         <property name="MSBuildToolsPath32" value="$([MSBuild]::GetToolsDirectory32())" />
         <property name="MSBuildToolsPath64" value="$([MSBuild]::GetToolsDirectory64())" />
         <property name="MSBuildSDKsPath" value="$([MSBuild]::GetMSBuildSDKsPath())" />
-        <property name="FrameworkSDKRoot" value="$(Registry:HKEY_LOCAL_MACHINE\SOFTWARE\Microsoft\Microsoft SDKs\NETFXSDK\4.6.1@InstallationFolder)" />
+        <property name="FrameworkSDKRoot" value="$(Registry:HKEY_LOCAL_MACHINE\SOFTWARE\Microsoft\Microsoft SDKs\NETFXSDK\4.7.2@InstallationFolder)" />
```

Comment:
Unrelated to this PR, I think this might need to use the 32-bit view just like toolspath below.